test/sequential/rolldown.updateL1FromL1.test.ts

```diff
@@ -4,19 +4,82 @@
  */
 
 import { EthUser } from "../../utils/EthUser";
-import { SequencerStaking } from "../../utils/rollDown/SequencerStaking";
+import {
+  ChainName,
+  SequencerStaking,
+} from "../../utils/rollDown/SequencerStaking";
 import { L2Update, Rolldown } from "../../utils/rollDown/Rolldown";
-import { BN_MILLION, BN_THOUSAND, BN_ZERO, signTx } from "gasp-sdk";
+import {
+  BN_HUNDRED,
+  BN_MILLION,
+  BN_TEN_THOUSAND,
+  BN_THOUSAND,
+  BN_ZERO,
+  signTx,
+} from "gasp-sdk";
 import { getApi, initApi } from "../../utils/api";
-import { setupUsers } from "../../utils/setup";
+import { setupApi, setupUsers } from "../../utils/setup";
 import {
   expectExtrinsicFail,
   expectExtrinsicSucceed,
   stringToBN,
   waitForNBlocks,
 } from "../../utils/utils";
-import { Keyring } from "@polkadot/api";
+import { ApiPromise, Keyring } from "@polkadot/api";
 import { testLog } from "../../utils/Logger";
+import { Sudo } from "../../utils/sudo";
+import { Assets } from "../../utils/Assets";
+import { FoundationMembers } from "../../utils/FoundationMembers";
+import { Maintenance } from "../../utils/Maintenance";
+import {
+  expectMGAExtrinsicSuDidSuccess,
+  filterAndStringifyFirstEvent,
+  waitForEvents,
+  waitNewBlock,
+} from "../../utils/eventListeners";
+import { BN } from "@polkadot/util";
+import { AssetWallet, User } from "../../utils/User";
+import { getAssetIdFromErc20 } from "../../utils/rollup/ethUtils";
+
+async function checkAndSwitchMnModeOff() {
```

Comment:
Typo! MmModeOff, 
all is about MaintenaceMode :) not Mn

Reply:
There was MmModeOff here, but I renamed it to MmOff